docs/sync.md

```diff
@@ -42,21 +42,21 @@ Now restart espanso and you should be ready to go!
 By default, the espanso configuration folder resides in this folder (change "user" with your username):
```

Comment:
Hey @kylebradshaw,
Thank you for your contribution! Could you please remove this part from the paragraph?
```
(change "user" with your username)
```
as now there is no user anymore :)

Reply:
👍